### PR TITLE
Add sponsorship badge

### PR DESCRIPTION
**Pull request by bronzdoc**
Add sponsorship badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,3 +191,9 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 ## Contributing
 
 Bug reports and pull requests are welcome on GitHub at https://github.com/fastruby/skunk/issues.
+
+## FastRuby.io
```
Comment (etagwerker) [inline README.md:195]:
```suggestion
## Sponsorship
```

File: README.md [hunk 2]
```diff
@@ -191,3 +191,9 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
...
+
+![fastruby](https://github.com/fastruby/skunk/raw/master/fastruby-logo.png)
```
Comment (etagwerker) [inline README.md:197]:
```suggestion
![FastRuby.io | Rails Upgrade Services](https://github.com/fastruby/skunk/raw/master/fastruby-logo.png)
```

File: README.md [hunk 3]
```diff
@@ -191,3 +191,9 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
...
+
+`skunk` is maintained and funded by FastRuby.io, inc. The names and logos for FastRuby.io are trademarks of FastRuby.io, inc.
```
Comment (etagwerker) [inline README.md:199]:
```suggestion
`skunk` is maintained and funded by [FastRuby.io](https://fastruby.io). The names and logos for FastRuby.io are trademarks of The Lean Software Boutique LLC.
```

File: README.md (hunk 2 above)
Comment (etagwerker) [reply README.md:197]: @bronzdoc Any thoughts about this? Our brand is "FastRuby.io" or "FastRuby.io | Rails Upgrade Services" -- source: https://fastruby.github.io/styleguide/

Comment (bronzdoc) [reply README.md:197]: @etagwerker I like "FastRuby.io | Rails Upgrade Services"

**Pull request merged**